sdks/go/examples/large_wordcount/large_wordcount.go

```diff
@@ -0,0 +1,429 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// large_wordcount is an example that demonstrates a more complex version
+// of a wordcount pipeline. It uses a SplittableDoFn for reading the
+// text files, then uses a map side input to build sorted shards.
+//
+// This example, large_wordcount, is the fourth in a series of five
+// successively more detailed 'word count' examples. You may first want to
+// take a look at minimal_wordcount and wordcount.
+// Then look at debugging_worcount for some testing and validation concepts.
+// After you've looked at this example, follow up with the windowed_wordcount
+// pipeline, for introduction of additional concepts.
+//
+// Basic concepts, also in the minimal_wordcount and wordcount examples:
+// Reading text files; counting a PCollection; executing a Pipeline both locally
+// and using a selected runner; defining DoFns.
+//
+// New Concepts:
+//
+//   1. Using a SplittableDoFn transform to read the IOs.
```

Comment:
Someone walking through these examples may not know what a SplittableDoFn is - it might be worth either:
a) Providing a brief description
b) Linking to https://beam.apache.org/blog/splittable-do-fn/
c) both

Reply:
Alternately (preferably?), you could call out where you're using it below and add a description there like you do with side inputs - I think that is really well described

Reply:
Great call! Done and done.